resources/views/admin/emails/example.blade.php

```diff
@@ -0,0 +1,13 @@
+@extends('admin.emails.main')
```

Comment:
I think that emails should me in separate folder. They are not related to frontend.
SO better to use folder structure like `emails.admin.main`